nano/scripts/nano/nano_state_pda.js

```diff
@@ -0,0 +1,57 @@
+NanoStatePDAClass.inheritsFrom(NanoStateClass);
+var NanoStatePDA = new NanoStatePDAClass();
```

Comment:
It's the pre-existing convention for states. Technically just `new NanoStatePDAClass();` would work, but the other class does this.